docs/cimpl6ToolingReference.md

```diff
@@ -29,24 +28,23 @@ The CIMPL Tooling, also called SHR-CLI (Standard Health Record Command Line Inte
 * [FHIR Examples](#fhir-examples) that are to be included in the IG, and
 * A [Package List](#package-list-file) that has information required for building the IG.
 
-### Processing Sequence
+The inputs are processed in the following sequence:
 
 * The user [issues a build command](#executing-shr-cli) through the command-line interface (CLI) to launch SHR-CLI.
 * The CIMPL tooling imports definitions from CIMPL files (class, value set, and map files). SHR-CLI [reports any errors](#addressing-error-messages) in the CIMPL definitions.
 * Continuing the build process, SHR-CLI selects a subset of the data models to include in the IG, according to the [filter strategy](#filter-strategy-configuration-parameters).
 * To complete the build process, SHR-CLI exports the selected CIMPL definitions into desired formats, such as FHIR profiles, data dictionaries, etc. The exports can be selected through [command line options](#executing-shr-cli). 
 * The user [issues a separate command](#creating-the-implementation-guide) to produce the IG.
 
-### Outputs
-
-Depending on the configuration options selected, the SHR-CLI produces one or more of the following outputs:
+SHR-CLI produces one or all of the following outputs, depending on configuration parameters:
 
 * [FHIR Profiles, Extensions, Value Sets](#fhir-export) that form the core content of the IG,
 * A [Logical Model](#logical-model-export) corresponding to the CIMPL class definitions, expressed as FHIR StructureDefinitions,
 * [JSON Schema](#json-schema-export) for the profiles defined by the IG,
 * A [Data Dictionary](#data-dictionary-export) that lists the Must-Support data elements in the IG, as well as value sets and value set members,
 * [Model Documentation](#model-documentation-export) in the form of a [Javadoc-like](#https://docs.oracle.com/javase/7/docs/api/) browser that allows one to see the hierarchical class relationships in the logical model.
```

Comment:
broken link. Remove the '#' from '#https://docs.oracle.com/javase/7/docs/api/'

Reply:
fixed.